### PR TITLE
[Grid offsetting] Breaks out the margin in the scss loop

### PR DESCRIPTION
**Pull request by MikeBallan**

### CODE REVIEW

File: scss/_grid.scss [hunk 1]
```diff
@@ -63,9 +67,13 @@ Tablet (Smaller than 959px)
 .row                                            { width:768px; }
 .row .row                                       { margin:0 -14px; }
 
+.grid-1, .grid-2, .grid-3, .grid-4,
+.grid-5, .grid-6, .grid-7, .grid-8,
+.grid-9, .grid-10, .grid-11, .grid-12           { margin:0 14px; }
+
 @for $i from 1 through 12                       {
-    .grid-#{$i}                                 { width: #{36 + 64 * ($i - 1)}px; margin:0 14px; }
-    .grid-#{$i}-tablet                          { width: #{36 + 64 * ($i - 1)}px; margin:0 14px; }
```
Comment (catchamonkey) [inline scss/_grid.scss]: you don't need the margin at all for the ```-tablet``` selectors?

Comment (MikeBallan) [inline scss/_grid.scss]: No as this is a modifier so it will all ready have the margin setup on it.

Comment (catchamonkey) [inline scss/_grid.scss]: Ah yes I misread where we were here, this is for mobile overrides on size like;

```
<div class="grid-2 grid-4-tablet">
```

Comment (MikeBallan) [inline scss/_grid.scss]: Yer that's it, also i guess you could merge the two together too as there the same code right?

Comment (catchamonkey) [inline scss/_grid.scss]: yeah just comma separate if you want, the resulting CSS will be cleaner I would imagine, not sure how intelligent the SCSS compiler is when seeing rules that are the same.

**Pull request merged**